app/screens/code/code.js

```diff
@@ -56,12 +56,13 @@ export default class Code extends React.PureComponent {
         const style = getStyleSheet(this.props.theme);
 
         const numberOfLines = this.countLines(this.props.content);
-        let lineNumbers = '1';
+        const lineNumbers = '1';
```

Comment:
Wondering if this should be 
```const lineNumbers = ['1']; ```
like in text_preview?

Reply:
Thanks for finding it. I missed it while copying it from my local repo.